src/h5web/App.tsx

```diff
@@ -18,11 +19,14 @@ function App(): JSX.Element {
         <ReflexElement className={styles.explorer} flex={0.3} minSize={250}>
           <Explorer
             filename="water_224.h5"
-            metadata={mockMetadata as HDF5Metadata}
+            metadata={mockMetadata as MockHDF5Metadata}
             onSelect={link => {
               setSelectedLink(link);
 
-              if (link.collection === HDF5Collection.Datasets) {
```

Comment:
A link can now be of multiple types, so I have to use a type guard `isHardLink` before assuming that `collection` is available.